packages/web-client/app/components/card-pay/balance-display/index.hbs

```diff
@@ -0,0 +1,39 @@
+{{#let (eq @size "small") (eq @size "large") as |isSmall isLarge|}}
+  <div class="balance-display" ...attributes>
+    {{#if @icon}}
+      {{svg-jar @icon class=(if isLarge "balance-display__icon-lg" "balance-display__icon") role="presentation"}}
+    {{/if}}
+    <div>
+      {{#if @label}}
+        <div class="balance-display__text-sm">{{@label}}</div>
+      {{/if}}
+      {{#if @name}}
+        <div
+          class={{if isSmall "balance-display__name-sm" "balance-display__name"}}
+          data-test-balance-display-name
+        >
+          {{@name}}
+        </div>
+      {{/if}}
+      {{#if @balance}}
+        <div class={{cn
+          balance-display__text=(not (or isSmall isLarge))
+          balance-display__text-lg=isLarge
+          balance-display__text-sm=isSmall
+        }} data-test-balance-display-amount>
+          {{format-token-amount @balance}} {{@symbol}}
+        </div>
+        {{#if @displayUsdBalance}}
```

Comment:
Many uses of usd balance use an approximate sign, which might be important:

![image](https://user-images.githubusercontent.com/39579264/122726820-f57ed200-d2a8-11eb-99b8-9e1a785fdfa0.png)

Reply:
Chris said to remove all approximate signs during one of the meetings. We can add it back if needed